src/sentry/integrations/vsts/notify_action.py

```diff
@@ -4,27 +4,27 @@
 
 from django import forms
 
-from sentry.rules.actions.base import IntegrationEventAction
+from sentry.rules.actions.base import TicketEventAction
+from sentry.models import ExternalIssue
 
 logger = logging.getLogger("sentry.rules")
 
 
 class AzureDevopsNotifyServiceForm(forms.Form):
     # TODO 2.0 Add form fields.
-
     def __init__(self, *args, **kwargs):
         super(AzureDevopsNotifyServiceForm, self).__init__(*args, **kwargs)
 
     def clean(self):
         return super(AzureDevopsNotifyServiceForm, self).clean()
 
 
-class AzureDevopsCreateTicketAction(IntegrationEventAction):
+class AzureDevopsCreateTicketAction(TicketEventAction):
     form_cls = AzureDevopsNotifyServiceForm
     label = u"TODO Create a {name} AzureDevops workitem"
-    prompt = "Create a AzureDevops workitem"
+    prompt = "Create an Azure DevOps work item"
     provider = "vsts"
-    integration_key = "project"
+    integration_key = "ado_integration"
```

Comment:
@ceorourke can we call it `vsts_integration` or `azure_integration`? I don't think we've used `ado` anywhere else.